Toggl.Giskard/Views/Calendar/CalendarLayoutManager.LayoutSate.cs

```diff
@@ -0,0 +1,71 @@
+using System.Collections.Generic;
+using Android.Support.V7.Widget;
+using Android.Views;
+
+namespace Toggl.Giskard.Views.Calendar
+{
+    public partial class CalendarLayoutManager
+    {
+        private const int TOWARDS_THE_END = 1;
```

Comment:
```suggestion
        private const int TowardsTheEnd = 1;
```
Same for the next two lines 🙂